e2e/cypress/integration/channel/message_shortlinking_spec.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+describe('Message', () => {
+    beforeEach(() => {
+        // # Login as "user-1" and go to /
+        cy.apiLogin('user-1');
+
+        cy.visit('/');
+    });
+
+    it('M17451 Channel shortlinking still works when placed in brackets', () => {
+        // # Post shortlink of channel
+        const shortLink = '(~saepe-5)';
+        const longLink = '~doloremque';
+
+        cy.postMessage(shortLink);
+        cy.postMessage(shortLink + ' some more other message');
```

Comment:
What's the intent with these other messages after the first one if you only verify the last message?

Reply:
No intent, I was using them too see if it worked as intended. Removed 👍